pkg/manager/member/pd_member_manager.go

```diff
@@ -756,6 +762,52 @@ func getPDConfigMap(tc *v1alpha1.TidbCluster) (*corev1.ConfigMap, error) {
 	return cm, nil
 }
 
+func (pmm *pdMemberManager) checkNotJoinClusterNode(tc *v1alpha1.TidbCluster, set *apps.StatefulSet, pdStatus map[string]v1alpha1.PDMember) error {
```

Comment:
I'm not mean to get into naming issues, but `checkSomething` usually does not mutate the input pointers, and `Node` is ambiguous with Kubernetes Node, maybe `collectUnjoinedMembers` could be more accurate